tests/wpt/metadata/fetch/content-length/parsing.window.js.ini

```diff
@@ -41,16 +41,10 @@
   [Content-Length%3A%20]
     expected: FAIL
 
-  [Input: "Content-Length: 42,42". Expected: 42.]
-    expected: FAIL
-
   [Input: "Content-Length: 42\\r\\nContent-Length: 42,42". Expected: 42.]
     expected: FAIL
 
-  [Input: "Content-Length: 30,30". Expected: 30.]
-    expected: FAIL
-
-  [Input: "Content-Length: 30\\r\\nContent-Length: 30,30". Expected: 30.]
+  [Input: "Content-Length: 030,30". Expected: network error.]
```

Comment:
We'll need a line that looks like this as well:
`[Input: "Content-Length: 030, 30". Expected: network error.]` (note the space after `,`)